recipes-core/aziot-edged/aziot-edged.inc

```diff
@@ -27,29 +28,28 @@ do_install () {
 
     # Data dir
     install -d ${D}${localstatedir}/lib
-    install -d ${D}${localstatedir}/log
     install -d ${D}${localstatedir}/lib/aziot
-    install -d ${D}${localstatedir}/log/aziot
     install -d -m 755 -o iotedge -g iotedge ${D}${localstatedir}/lib/aziot/edged
-    install -d -m 755 -o iotedge -g iotedge ${D}${localstatedir}/log/aziot/edged
 
     install -d ${D}${systemd_unitdir}/system
     install -m 644 ${WORKDIR}/aziot-edged.service ${D}${systemd_unitdir}/system
     install -m 644 ${WORKDIR}/git/edgelet/contrib/systemd/debian/aziot-edged.workload.socket ${D}${systemd_unitdir}/system
     install -m 644 ${WORKDIR}/git/edgelet/contrib/systemd/debian/aziot-edged.mgmt.socket ${D}${systemd_unitdir}/system
 
+    #Permissions and ownership for /var/run/iotedge
```

Comment:
Is this comment misleading?

Reply:
`and` and `for` are shown as keywords. Is that what you are inferring?

Reply:
No. The path in the comment. _I now believe it is correct_. I didn't have access to a working example at the time I made the comment. Now I do.

Reply:
The path is correct?